server/scripts/dbPopulate.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose')
+const tripSchema = require('../models/TripModel')
+const stageSchema = require('../models/StageModel')
+const userSchema = require('../models/UserModel')
+const { ObjectId } = require('mongodb')
+
+const conString =
+  'mongodb+srv://bugstorm1000:GqHIshuc7OuDPDd1@cluster0.4bflh41.mongodb.net/BugStormDEV_Rithin?retryWrites=true&w=majority'
```

Comment:
instead of BugStormDEV_Rithin, we would ideally want process db from `.env`

Reply:
i dont think we should expose the connection string like this, it has the username and password in it. you should us config and do sth like: mongoose.connect(config.mongo.uri)